src/lib/permissions/permissionsApiClient.class.ts

```diff
@@ -0,0 +1,27 @@
+import type { PermissionsApiClientConstructor } from './clients/interfaces';
+import { ForumPermissionsHttpClient } from './forums/client/forumPermissionsHttpClient';
+import type { PremiumForumPermissionsClient } from './forums/client/interfaces';
+import type { PremiumPermissionsClient } from './interfaces';
+import type { PremiumPagePermissionsClient } from './pages/client/interfaces';
+import { PagePermissionsHttpClient } from './pages/client/pagePermissionsHttpClient';
+import type { PremiumProposalPermissionsClient } from './proposals/client/interfaces';
+import { ProposalPermissionsHttpClient } from './proposals/client/proposalPermissionsHttpClient';
+import type { PremiumSpacePermissionsClient } from './spaces/client/interfaces';
+import { SpacePermissionsHttpClient } from './spaces/client/spacePermissionsHttpClient';
+
+export class PermissionsApiClient implements PremiumPermissionsClient {
```

Comment:
@mattcasey took this out of "clients folder" and bumped it up a level to root of permissions, since it's one of main exports that consumes rest of lib